### PR TITLE
Edit priya

### PR DESCRIPTION
**Pull request by priyanayar**
Hello @moriohara : Added documentation to use User defined certificates to create SBS instance for sbs related PSRIT. Please review. Thank you.

### CODE REVIEW

**Comment by priyanayar** [comment]
@moriohara : Thank you for your feedback. Incorporated them and please review.

File: README.md [hunk 1]
```diff
@@ -155,12 +155,12 @@ ibmcloud plugin install hpvs
...
          ```
       3. Create a the Hyper Protect Virtual Servers instance by using the `ibmcloud hpvs instance-create command`.  
-          ```
-          ibmcloud hpvs instance-create docker.io-ibmzcontainers-acrux-dev1 lite-s syd05 --rd-path secure_build.asc --image-tag 1.3.0 $CERT_ENV
-          ```
-          Continue to step #6.          
-          Note:-     
-         - Follow the best practices of certificate management. 
+         ```
+         ibmcloud hpvs instance-create docker.io-ibmzcontainers-acrux-dev1 lite-s syd05 --rd-path secure_build.asc --image-tag 1.3.0 $CERT_ENV
```
Comment (abhiramkulkarni) [inline README.md:159]: tag should be 1.3.0.1 ?

**Comment by priyanayar** [comment]
Updated readme.md with latest SBS image tag of 1.3.0.1

**Review by abhiramkulkarni — APPROVED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (priyanayar) [reply README.md]: Updated! All instances.

**Review by moriohara — COMMENTED** [review]
LGTM

**Comment by priyanayar** [comment]
Thank you @moriohara

**Pull request merged**